tests/lib/wifi.py

```diff
@@ -177,3 +178,165 @@ def uciSetWifiSecurity(board, vap_iface, security):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_password(device, board, iface_ip, mib_name, index, set_type, password):
+    mib_out = snmp_mib_set(device, board,iface_ip,mib_name,index,set_type,password)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_password(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device, board,iface_ip,mib_name ,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_channel_util(device, board, iface_ip, mib_name, index, set_type, channel_util):
+    mib_out = snmp_mib_set(device, board,iface_ip,mib_name,index,set_type,channel_util)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_channel_util(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device, board,iface_ip,mib_name ,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_operating_mode(device, board, iface_ip, mib_name, index, set_type, operating_mode):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,operating_mode)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_operating_mode(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name ,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_bandwidth(device, board, iface_ip, mib_name, index, set_type, bandwidth):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,bandwidth)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def get_bandwidth(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_channel_no(device, board, iface_ip, mib_name, index, set_type, channel_no):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,channel_no)
+    if mib_out != None:
+       return mib_out
+    else:
+       return None
+
+def get_channel_no(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
+
+def set_broadcast(device, board, iface_ip, mib_name, index, set_type, broadcast):
+    mib_out = snmp_mib_set(device,board,iface_ip,mib_name,index,set_type,broadcast)
+    if mib_out != None:
+       return mib_out
+    else:
+       return None
+
+def get_broadcast(device, board, iface_ip, mib_name, index):
+    mib_out = snmp_mib_get(device,board,iface_ip,mib_name,index)
+    if mib_out != None:
+        return mib_out
+    else:
+        return None
```

Comment:
I don't think we need these functions. I think the device class handler for wifi_snmp() can just call these. These might not be generic enough for other board types anyways so we will just let the device classes handle it all.

Reply:
Sure. Will dissolve this function and will call snmp set and get directly in the device class handler for wifi_snmp()